ruby/vampires.rb

```diff
@@ -0,0 +1,103 @@
+puts "How many employees will be processed?"
+n = gets.chomp.to_i
+
+while n > 0
+  puts "Thank you for taking our new employee survey!"
+  
+  # Questions to collect data from the employee.
+
+  puts "What is your name?"
+  name = gets.chomp
+  
+  puts "How old are you, or how old will you be turning this year?"
+  age = gets.chomp.to_i
+  
+  puts "What year were you born?"
+  birth_year = gets.chomp.to_i
+  by_age = 2017 - birth_year
+  
+  puts "Our company cafeteria serves garlic bread. Should we order some for you? (y/n)"
+  gbread = gets.chomp
+  
+  if gbread == "y"
```

Comment:
There is some refactoring that could be done, but logic is good here.  For instance: 
```
if garlic_bread.downcase == "yes" || garlic_bread.downcase == "y"
   wants_garlic = true
else
   wants_garlic = false
end
```
can be refactored to:
```
wants_garlic = garlic_bread.downcase == "yes" || garlic_bread.downcase == "y"
```
If `garlic_bread.downcase` is neither of those answers, `wants_garlic` will be set to false.